lib/hsa/kernels/rfi_chime_input_sum.cl

```diff
@@ -55,16 +66,35 @@ rfi_chime_inputsum(
         uint address = gy + gz * gy_size;
         if (n * N == 0){
             output[address] = -1.0;
+            output_var[address] = -1.0;
             output_mask[address] = 1;
         } else {
             float new_sq_power_across_input = sq_power_across_input[0] * cf * cf;
             float SK = ((n + 1) / (n - 1)) * ((new_sq_power_across_input / (n * N)) - 1);
+
+            // Calculate the truncation bias correction for the SK value
+            const float var = var_across_input[0] * one_over_256;
+            const float var2 = var * var;
+            const float var_sqrt = sqrt((double)var);
+            const float sk_correction = bias_coeffs[0] * var2 * var_sqrt + 
+                                        bias_coeffs[1] * var2 +
+                                        bias_coeffs[2] * var * var_sqrt +
+                                        bias_coeffs[3] * var +
+                                        bias_coeffs[4] * var_sqrt +
+                                        bias_coeffs[5]; 
```

Comment:
Two comments here. First I think you'd be better of reversing the order of `bias_coeffs`. There's two reasons for that i) there's a direct correspondence between the index and the power you are raising the rms to; ii) if we need to add terms to make things more accurate (i.e. a 6th order term) you don't need to rewrite this expression.

Second, evaluating like this is a little clunky. You might consider writing this with a for loop (and thus building up the powers as you go), or writing it as a nested series for multiply-add statements, e.g. (with the ordering of bias_coeffs reverse):
```C
// either ...
float sk_correction = 0;
float rms_pow = 1;
for (int i = 0; i < NCOEFF; i++) {
    sk_correction += bias_coeffs[i] * rms_pow;
    rms_pow *= rms;
}
/// ... or ...
const float sk_correction = bias_coeffs[0] + rms * (bias_coeffs[1] + rms * (bias_coeffs[2] + rms * (bias_coeffs[3] + rms * ...)
```